src/bin/club-story

```diff
@@ -36,6 +37,9 @@ program
     .option('-y, --type [name]', 'Update type of story', '')
     .option('--from-git', 'Fetch story parsed by ID from current git branch')
     .option('--git-branch', 'Create git branch from story slug <type>-<id>-<title>')
+    .option('--git-branch-standard',
+        'Create git branch from story slug <mention-name>/ch<id>/<title> ' +
```

Comment:
To better format this in the `--help` output, can we add a newline and tabs here?

```js
.option('--git-branch-standard',
     'Create git branch from story slug <mention-name>/ch<id>/<title> \n' +
     '\t\t\t\tas required by Clubhouse's Git integration: https://bit.ly/2RKO1FF')
```

Which will output:

```sh
    --git-branch              Create git branch from story slug <type>-<id>-<title>
    --git-branch-standard     Create git branch from story slug <mention-name>/ch<id>/<title> 
    				as required by Clubhouse's Git integration: https://bit.ly/2RKO1FF
```